src/main/java/duke/helper/Ui.java

```diff
@@ -0,0 +1,86 @@
+package duke.helper;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Ui class that handles all the interactions with the users
+ */
+public class Ui {
+    private Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public static String showErrorMsg(String desc) {
+        return String.format("%s commands need to be followed by an integer!\n", desc.split(" ")[0]);
+    }
+
+    public static String showErrorMsg(int size) {
+        return String.format("Sorry but there are only %d tasks stored!\n", size);
+    }
+
+    /**
+     * Prints the exit message and close the scanner
+     */
+    public String showExit() {
+        scanner.close();
+        return "Bye. Hope to see you again soon!\n";
+    }
+
+    /**
+     * Prints the delete message when a task is removed
+     *
+     * @param task task to be removed
+     * @param size number of remaining tasks
+     */
+    public static String showDelete(Task task, int size) {
```

Comment:
These can be output by a Task -> DeleteTask class that is responsible for outputting the relevant UI message to the UI. The UI then need only have a single method that parses string messages and outputs them in the desired format, or even be changed to output UI elements in the GUI.